benchmark/aws/instance.py

```diff
@@ -129,13 +129,18 @@ def _get_vpc_info(self, client):
                 vpc_ids += [x['VpcId']]
 
         subnet_ids = {}
-        for x in client.describe_subnets()['Subnets']:
+        for x in client.describe_subnets(
+            Filters=[{
+                'Name': 'cidr-block', 
+                'Values': [ self.settings.subnet_cidr ]
+            }]
+            )['Subnets']:
```

Comment:
@asonnino this finds the right subnet in the VPC, by matching on its CIDR. 
Without this, if the VPC had more than one subnet, the for loop would end up just picking the last subnet returned by `describe_subnets()`